scripts/integration-tests.sh

```diff
@@ -284,10 +284,15 @@ if [[ "$STACK_CONFIG" == *"docker:"* && "$COLLECT_ONLY" == false ]]; then
     docker stop "$container_name" 2>/dev/null || true
     docker rm "$container_name" 2>/dev/null || true
 
+    # Configure telemetry collector port shared between host and container
```

Comment:
do you not need to run the docker with this additional port mapped or something?

Reply:
ah I guess we run it with `--network host` -- which of course doesn't quite work on a mac :)